README.md

```
@@ -297,6 +297,7 @@ linux-wallpaperengine --set-property bloom=1 2370927443
 ## 🧪 Wayland & X11 Support
 
 - **Wayland**: Works with compositors that support `wlr-layer-shell-unstable`.
+- **KDE Wayland**: Users of KDE Plasma under Wayland that may wish to make use of the "pause on fullscreen" features may do so but require the installation of an additional script found [here](https://github.com/LS-FCEFyN/Maximize-Detector)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)